data/org.freedesktop.impl.portal.Background.xml

```diff
@@ -28,6 +28,51 @@
   -->
   <interface name="org.freedesktop.impl.portal.Background">
 
+    <!--
+      GetAppState:
+      @apps: an array with information about running apps
+
+      Gets information about running apps. Each entry has
+      an application ID as key. The returned values are of
+      type u and have the following meaning:
+      <simplelist>
+        <member>0: Background (no open window)</member>
+        <member>1: Running (at least one open window)</member>
+        <member>2: Active (in the foreground)</member>
+      </simplelist>
+    -->
+    <method name='GetAppState'>
+      <arg type="a{sv}" name="apps" direction="out"/>
+    </method>
+
+    <!--
+      NotifyBackground:
+      @handle: Object path for the #org.freedesktop.impl.portal.Request object representing this call
+      @app_id: App id of the application
+      @name: A display name for the application
+      @response: Numeric response, not used
+      @results: Vardict with results of the call
+
+      Notifies the user that an application is running in the background.
+
+      The following results get returned via the @results vardict:
+      <variablelist>
+        <varlistentry>
+          <term>allow b</term>
+          <listitem><para>
+            TRUE to allow the app to run in the background, FALSE to stop it.
```

Comment:
Should we allow replying "allow once"?

Reply:
Needs some cleanup for the application tracking first, to avoid notifying multiple times